hitbtc.go

```diff
@@ -286,18 +290,28 @@ func (b *HitBtc) PlaceOrder(requestOrder Order) (responseOrder Order, err error)
 	payload["quantity"] = fmt.Sprintf("%.8f", requestOrder.Quantity)
 	payload["price"] = fmt.Sprintf("%.8f", requestOrder.Price)
 
-	r, err := b.client.do("PUT", "order/"+requestOrder.ClientOrderId, payload, true)
+	method := "POST"
+	resource := "order"
+
+	if requestOrder.ClientOrderId != "" {
+		method = "PUT"
+		resource = fmt.Sprintf("%s/%s", resource, requestOrder.ClientOrderId)
+	}
+
+	r, err := b.client.do(method, resource, payload, true)
 	if err != nil {
 		return
 	}
+
 	var response interface{}
 	if err = json.Unmarshal(r, &response); err != nil {
 		return
-	}
-	if err = handleErr(response); err != nil {
+	} else if err = handleErr(response); err != nil {
```

Comment:
if the above branch always returns, no need the "else", just "if"

Reply:
Thx will correct